src/content/en/updates/2020/11/cvd.md

```diff
@@ -0,0 +1,478 @@
+
+Let’s see how we can make this less intrusive. There’s two parts to this solution that we need to hide: 1) the CSS style with the `filter` property, and 2) the SVG filter definition, which is currently part of the DOM.
+
+```html
+<!-- Part 1: the CSS style with the filter property -->
+<style>
+  :root {
+    filter: url(#deuteranopia);
+  }
+</style>
+<!-- Part 2: the SVG filter definition -->
+<svg>
+  <filter id="deuteranopia">
+    <feColorMatrix values="0.367  0.861 -0.228  0.000  0.000
+                           0.280  0.673  0.047  0.000  0.000
+                          -0.012  0.043  0.969  0.000  0.000
+                           0.000  0.000  0.000  1.000  0.000">
+  </filter>
+</svg>
+```
+
+### Avoiding the in-document SVG dependency
+
+Let’s start with part 2: how can we avoid adding the SVG to the DOM? One idea is to move it to a separate SVG file. We can copy the `<svg>…</svg>` from the above HTML and save it as `filter.svg` — but we need to make some changes first! Inline SVG in HTML follows the HTML parsing rules. That means you can get away with things like [omitting quotes around attribute values in some cases](https://mathiasbynens.be/notes/unquoted-attribute-values). However, SVG in separate files is supposed to be valid XML — and XML parsing is way more strict than HTML. Here’s our SVG-in-HTML snippet again:
+
+```html
+<svg>
+  <filter id="deuteranopia">
+    <feColorMatrix values="0.367  0.861 -0.228  0.000  0.000
+                           0.280  0.673  0.047  0.000  0.000
+                          -0.012  0.043  0.969  0.000  0.000
+                           0.000  0.000  0.000  1.000  0.000">
+  </filter>
+</svg>
+```
+
+To make this valid standalone SVG (and thus XML), we need to make two changes. Can you guess which?
+
+<!-- ```xml
+<svg xmlns="http://www.w3.org/2000/svg">
+  <filter id="deuteranopia">
+    <feColorMatrix values="0.367  0.861 -0.228  0.000  0.000
+                           0.280  0.673  0.047  0.000  0.000
+                          -0.012  0.043  0.969  0.000  0.000
+                           0.000  0.000  0.000  1.000  0.000" />
+  </filter>
+</svg>
+``` -->
+<pre class="prettyprint">
+&lt;svg <strong>xmlns=&quot;http://www.w3.org/2000/svg&quot;</strong>&gt;
+  &lt;filter id=&quot;deuteranopia&quot;&gt;
+    &lt;feColorMatrix values=&quot;0.367  0.861 -0.228  0.000  0.000
+                          0.280  0.673  0.047  0.000  0.000
+                          -0.012  0.043  0.969  0.000  0.000
+                          0.000  0.000  0.000  1.000  0.000&quot; <strong>/&gt;</strong>
+  &lt;/filter&gt;
+&lt;/svg&gt;
+</pre>
```

Comment:
On the preview, the WebFu styles have low-contrast text except for the highlighted parts, so let's try highlighting it as normal, and just adding `<mark>` where applicable:

```suggestion
<pre class="prettyprint"><code class="html"><span class="tag"><span class="tag">&lt;svg </span></span><mark><span class="atn"><span class="atn">xmlns</span></span><span class="pun"><span class="pun">=</span></span><span class="atv"><span class="atv">"http://www.w3.org/2000/svg"</span></span></mark><span class="tag"><span class="tag">&gt;</span></span><span class="pln"><span class="pln"><br>&nbsp; </span></span><span class="tag"><span class="tag">&lt;filter</span></span><span class="pln"><span class="pln"> </span></span><span class="atn"><span class="atn">id</span></span><span class="pun"><span class="pun">=</span></span><span class="atv"><span class="atv">"deuteranopia"</span></span><span class="tag"><span class="tag">&gt;</span></span><span class="pln"><span class="pln"><br>&nbsp; &nbsp; </span></span><span class="tag"><span class="tag">&lt;feColorMatrix</span></span><span class="pln"><span class="pln"> </span></span><span class="atn"><span class="atn">values</span></span><span class="pun"><span class="pun">=</span></span><span class="atv"><span class="atv">"0.367 &nbsp;0.861 -0.228 &nbsp;0.000 &nbsp;0.000<br>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;0.280 &nbsp;0.673 &nbsp;0.047 &nbsp;0.000 &nbsp;0.000<br>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; -0.012 &nbsp;0.043 &nbsp;0.969 &nbsp;0.000 &nbsp;0.000<br>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;0.000 &nbsp;0.000 &nbsp;0.000 &nbsp;1.000 &nbsp;0.000"</span></span><mark><span class="tag"><span class="tag"> /&gt;</span></span></mark><span class="pln"><span class="pln"><br>&nbsp; </span></span><span class="tag"><span class="tag">&lt;/filter&gt;</span></span><span class="pln"><span class="pln"><br></span></span><span class="tag"><span class="tag">&lt;/svg&gt;</span></span><span class="pln"><span class="pln"><br></span></span></code></pre>
```